README.md

```diff
@@ -656,6 +656,25 @@ $ oc adm policy add-scc-to-user privileged -z awx
 
 Again, this is the most relaxed SCC that is provided by OpenShift, so be sure to familiarize yourself with the security concerns that accompany this action.
 
+#### Containers HostAliases Requirements
+
+Sometimes you might need to use HostAliases in web/task containers.
```

Comment:
Could you add a link to https://kubernetes.io/docs/tasks/network/customize-hosts-file-for-pods/ in the docs here?

Reply:
https://github.com/ansible/awx-operator/pull/1232/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5